src/rules/export.js

```diff
@@ -50,7 +50,13 @@ module.exports = function (context) {
         return
       }
       let any = false
-      remoteExports.forEach((v, name) => (any = true) && addNamed(name, node))
+      remoteExports.forEach((v, name) => (
+        (any = true) &&
```

Comment:
FWIW, `remoteExports` is more of a `Map` than an `Array`, so it has `forEach` but no `filter`. It has a `size`, but this size includes the `default` name.

I agree that the `(any = true)` assignment expression is a little laconic, but it gets the job done.